.github/workflows/test.yml

```diff
@@ -41,10 +41,8 @@ jobs:
             testsuite:    system-test
           - compiler:     clang
             testsuite:    test
-            asan:         asan
-          - compiler:     clang
-            testsuite:    test
-            ubsan:        ubsan
+            sanitizers:   sanitizers
+            cflags:       -g
```

Comment:
I changed https://github.com/ovn-org/ovn/blob/23e203a3f30bd57ef8d0def7ee2698e588782d0c/.ci/linux-build.sh#L19-L26

to 
```sh
function configure_ovn()
{
    configure_ovs $*

    export CFLAGS="${OVS_CFLAGS} ${OVN_CFLAGS}"
    ./boot.sh && ./configure $* || \
    { cat config.log; exit 1; }
}
```

but it doesn't help 😿 
I see `-O1 -fno-omit-frame-pointer -fno-common -g -Wno-error=unused-command-line-argument -g -O2`

Yes, I know it's hacking and if `CFLAGS` could be removed it would be better.